tests/tinderbox_scraper/test_tinderbox_scraper.py

```diff
@@ -142,6 +142,20 @@
      'target': 'mozilla-central-firefox-25.0a1.en-US.linux-i686.txt',
      'target_url': 'firefox/tinderbox-builds/mozilla-central-linux/'
                    '1374583608/firefox-25.0a1.en-US.linux-i686.txt'},
+    # -a firefox -t tinderbox -p win32 --extension=txt
+    {'args': {'application': 'firefox',
+              'extension': 'txt',
+              'platform': 'win32'},
+     'target': 'mozilla-central-firefox-25.0a1.en-US.win32.txt',
+     'target_url': 'firefox/tinderbox-builds/mozilla-central-win32/'
+                   '1374568307/firefox-25.0a1.en-US.win32.txt'},
```

Comment:
Your patch does not contain this txt file for the data subdir. As result Travis CI fails to run the tests. I assume you simply missed to add it.